### DISCUSSION TITLE
Checksum error gameboy carts

### DISCUSSION THREAD

**Discussion by tusmavan** [sanni/cartreader#662] [Need help]
im currently building the HW5 reader and got to the step to test the reader (i dont have a capacitor as i it was optional, and the led bulb isnt working aswell)
I keep getting a checksum error when trying to read GB/GBA carts. Im including the files it dumped. its supposed to be pokemon gold eu/usa.

i included the dumped files and a picture of my pcb.
thanks in advance for anyone trying to help

![pcb](https://user-images.githubusercontent.com/62932856/208323116-4af682c2-a378-4fda-8c91-9d33842f264f.jpg)

**Comment by sanni** [comment]
Reflow of your solder joints on the Mega 2560 Pro, bottom rows labeled A0-A15.
